sound/soc/sof/sof-client-probes-ipc4.c

```diff
@@ -152,6 +184,15 @@ static int ipc4_probes_deinit(struct sof_client_dev *cdev)
 	return sof_client_ipc_tx_message(cdev, &msg, NULL, 0);
 }
 
+/**
+ * ipc4_probes_points_info - retrieve list of active probe points
+ * @cdev:	SOF client device
+ * @desc:	Returned list of active probes
+ * @num_desc:	Returned count of active probes
+ * @return:		0 on success, negative error code on error
```

Comment:
not sure why we have an indentation issue here?

Reply:
Me neither. Will fix.